src/libAtomVM/term.h

```diff
@@ -749,6 +749,20 @@ static inline term term_make_maybe_boxed_int64(Context *ctx, avm_int64_t value)
     }
 }
 
+static inline size_t term_boxed_integer_size(avm_int_t value)
+{
+#if BOXED_TERMS_REQUIRED_FOR_INT64 == 2
+    if ((value < AVM_INT_MIN) || (value > AVM_INT_MAX)) {
+        return BOXED_INT64_SIZE;
```

Comment:
I think this will never be executed.

Reply:
I think it can be fixed by changing `static inline size_t term_boxed_integer_size(avm_int_t value)` to `
static inline size_t term_boxed_integer_size(avm_int64_t value)`

Reply:
See also: `term.h:static inline term term_make_maybe_boxed_int64(Context *ctx, avm_int64_t value`

Reply:
Please review changes, and thanks for the suggestions!